README.md

```diff
@@ -128,19 +128,18 @@ when downloading Go modules. See [Go environment variables](https://go.dev/ref/m
 
 ## Package managers
 
-Supported:
-
-* [gomod](#gomod)
-* [pip](#pip)
-* [npm](#npm)
-* [yarn](#yarn)
-* [bundler](#bundler)
-* [generic fetcher](#generic-fetcher)
-* [cargo](#cargo)
-
-Planned:
-
-* dnf
+| Package manager                                | Programming language |
```

Comment:
I'd like to limit the number of N/A (won't apply to generic ever) in the table, does "package ecosystem" sound better than _Programming language_?

Reply:
N/A is okay for generic, but not for RPMs, hence the "package ecosystem" suggestion. Once we adopt DNF as the backend the situation would repeat with N/A, whereas with ecosystem you simply put RPM in there. Similarly to e.g. DEB if we ever adopted Debian's package ecosystem, etc.